config/checkstyle-examples-checks.xml

```diff
@@ -99,6 +99,8 @@
 
   <module name="TreeWalker">
 
+    <module name="PackageDeclaration"/>
```

Comment:
We were missing package declarations in Examples.txt, this will cause problems as we activate more files.

Reply:
@nrmancuso, I have been stuck at a point where it doesn't allow me change the name of Example1.java to something elese stating the supression. But, if i keep it Example1.java, it complains about duplicate class

`Duplicate class found in the file '/Users/piyushkumarsadangi/Development/new/checkstyle/src/test/resources/com/puppycrawl/tools/checkstyle/checks/annotation/annotationusestyle/InputAnnotationUseStyleDifferentStyles.java'`

`[ERROR] [checkstyle] [ERROR] /Users/piyushkumarsadangi/Development/new/checkstyle/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/annotation/annotationusestyle/ExampleAnnotationStyle1.java:1: Example test input must declare exactly one class matching 'Example\d+' [exampleClassName]`

Will adding this module be of any help?

Reply:
@relentless-pursuit please keep discussion of changes in associated PRs